src/reconcile/rec_write.c

```diff
@@ -2798,8 +2795,15 @@ __wt_rec_cell_build_ovfl(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_KV *k
             WT_ERR(__wt_ovfl_reuse_add(session, page, addr, size, kv->buf.data, kv->buf.size));
     }
 
-    /* Set the callers K/V to reference the overflow record's address. */
-    WT_ERR(__wt_buf_set(session, &kv->buf, addr, size));
+    /*
+     * If the page stats feature flag is set, set the callers K/V to reference the combined address
+     * cookie which packs the page stats information with the overflow record's address. Otherwise,
+     * just set the callers K/V to reference the overflow record's address.
+     */
+    if (__wt_process.page_stats_2022 && WT_PAGE_STAT_VALID(&ovfl_ps))
```

Comment:
Here the same.

Reply:
Fixed here: [7fdd194](https://github.com/wiredtiger/wiredtiger/pull/8792/commits/7fdd194a5742efa7716451b29b381812ed336f2c#diff-5b8d4c04a02cd50ac6b7caf1af10ca16a9a4ea7aeae4e64d5e8bb1b6fa67e645)